frontend/packages/shared/src/components/altinnHeaderMenu/AltinnHeaderMenu.test.tsx

```diff
@@ -1,21 +1,28 @@
 import React from 'react';
 import { render as rtlRender, screen } from '@testing-library/react';
-import { AltinnHeaderMenu, AltinnHeaderMenuItem, IAltinnHeaderMenuProps } from './AltinnHeaderMenu';
+import { AltinnHeaderMenu, IAltinnHeaderMenuProps } from './AltinnHeaderMenu';
 import { MemoryRouter } from 'react-router-dom';
+import { TopBarMenuItem } from 'app-shared/types/TopBarMenuItem';
+import { TopBarMenu } from 'app-shared/enums/TopBarMenu';
+import { RoutePaths } from 'app-development/enums/RoutePaths';
+import { RepositoryType } from 'app-shared/types/global';
 
 describe('AltinnHeaderMenu', () => {
-  const mockMenu: AltinnHeaderMenuItem[] = [
+  const mockMenuItems: TopBarMenuItem[] = [
     {
-      key: 'key1',
-      link: <a href='link1'>Link1</a>,
+      key: TopBarMenu.About,
+      link: RoutePaths.Overview,
```

Comment:
Same as prev comment, we should avoid to import from app-development into shared package. 😊